app/services/manual_runner.rb

```diff
@@ -0,0 +1,39 @@
+class ManualRunner
+
+  def initialize(repo)
+    raise "Repo doesn't exist" unless Repo.exist?(repo)
+    @repo = repo
+  end
+
+  def run_commits_since(date)
+    fetched_commits = fetch_commits_since(date)
+    formated_commits = format_commits(fetched_commits)
+
+    CommitsRunner.run(formated_commits)
+  end
+
+  private
+
+  def fetch_commits_since(date)
+    Octokit.auto_paginate = true
```

Comment:
Does this paginate the response? Otherwise we might be fetching 2000 commits into memory.